spec/xcpretty/formatters/formatter_spec.rb

```diff
@@ -57,6 +57,11 @@ module XCPretty
 )
     end
 
+    it "formats linker warnings" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.